### ISSUE TITLE
[FEA] Support Dataframe/Series groupby level=0

### ISSUE THREAD

**Issue by mrocklin** [rapidsai/cudf#880]
It would be useful for dask dataframe integration in dataframe groupbys supported the `level=` keyword, at least with `level=0`, which signifies that we should be grouping on the index, as in the following Pandas example.

```python
In [1]: import pandas as pd

In [2]: df = pd.DataFrame({'x': [1, 2, 3]}, index=[0, 1, 0])

In [3]: df
Out[3]:
   x
0  1
1  2
0  3

In [4]: df.groupby(level=0).sum()
Out[4]:
   x
0  4
1  2
```

cc @thomcom @kkraus14

**Comment by kkraus14** [comment]
Fixed by #898

**Comment by mrocklin** [comment]
It would also be useful to have this for `Series` objects.  Reopening (or I'm happy to create a new issue)
